### ISSUE TITLE
Endpoint 'query_params' does not have a default value

### ISSUE THREAD

**Issue by rmccue** [WP-API/WP-API#860]
In `WP_JSON_Server::embed_links`, we use `$item['query_params']` without actually checking whether it exists. This means we get:

```
Notice: Undefined index: query_params in /vagrant/content/plugins/json-rest-api/lib/class-wp-json-server.php on line 370
```

**Comment by rmccue** [comment]
From #836.

**Comment by rmccue** [comment]
Fixed in #958.
